Remove DNS handling from istio (#615)
kyma-project/istio: docs/release-notes/1.3.2.md

## New features

> - Remove Istio IGW DNS handling. Module will not annotate `istio-ingressgateway` service with `dns.gardener.cloud/dnsnames` annotation for Gardener clusters. DNS will be handled in API Gateway with a Gardener's DNSEntry resource [#615](https://github.com/kyma-project/istio/pull/615).

Reviewer: ```suggestion
- Remove Istio IGW DNS handling. The module no longer uses the `dns.gardener.cloud/dnsnames` annotation to annotate the `istio-ingressgateway` service anymore in Gardener clusters. DNS is handled in the API Gateway module with a Gardener's DNSEntry resource [#615](https://github.com/kyma-project/istio/pull/615).
```
